common/src/main/java/com/wynntils/handlers/container/ContainerQueryHandler.java

```diff
@@ -141,7 +122,11 @@ public void onMenuForcefullyClosed(MenuEvent.MenuClosedEvent e) {
 
         // Server closed our container window. This should not happen
         // but if it do, report failure
-        raiseError("Server closed container");
+        if (e.getContainerId() == containerId) {
+            raiseError("Server closed container");
+        } else {
+            WynntilsMod.warn("Server closed container " + e.getContainerId() + " but we are querying " + containerId);
+        }
```

Comment:
This is purely experimental, but this allows even 10 queries to run after each other without problems, where as without this 2/3 repeated queries failed..